libtextsecure/message_receiver.js

```diff
@@ -1040,7 +1047,7 @@ MessageReceiver.prototype.extend({
       }
 
       // Exit early since the friend request reply will be a regular empty message
-      return;
+      return Promise.resolve();
```

Comment:
Same as above

Reply:
you can `return null` i think to fix the consistent return. @sachaaaaa thoughts?

Reply:
Yeah `return null` is fine !